src/Runner/Hook.php

```diff
@@ -449,10 +517,17 @@ private function getHookPlugins(): array
      */
     private function executeHookPluginsFor(string $method, ?Config\Action $action = null): void
     {
+        /** @var IO\Base $io */
+        $io = $this->io;
```

Comment:
This is for the benefit of PHPStan, since `getOption()` is not on the `Console\IO` interface.

Reply:
We will not need this if we are not using the `IO` to make decision inside `Runner`
See previous comment on `src/Runner/Hook.php`